### ISSUE TITLE
Human gut Bacteroidetes can utilize yeast mannan through a selfish mechanism.

### ISSUE THREAD

**Issue by pschloss** [SchlossLab/march_madness_2015#10]
Human gut Bacteroidetes can utilize yeast mannan through a selfish mechanism.
Cuskin F, Lowe EC, Temple MJ, Zhu Y, Cameron EA, Pudlo NA, Porter NT, Urs K, Thompson AJ, Cartmell A, Rogowski A, Hamilton BS, Chen R, Tolbert TJ, Piens K, Bracke D, Vervecken W, Hakki Z, Speciale G, Munōz-Munōz JL, Day A, Peña MJ, McLean R, Suits MD, Boraston AB, Atherly T, Ziemer CJ, Williams SJ, Davies GJ, Abbott DW, Martens EC, Gilbert HJ.
Nature. 2015 Jan 8;517(7533):165-9. doi: 10.1038/nature13995.
http://www.ncbi.nlm.nih.gov/pubmed/25567280

**Comment by pschloss** [comment]
One of the challenges we face in studying the human microbiota is deciphering what the bacteria are doing and how they do it. The paper by Cuskin and colleagues investigates the role of Bacteroides thetaiotamicron in degrading yeast derived mannans that are commonly found in breads and fermented foods. I picked this paper to read because my colleague, Eric Martens and his lab had a significant role in the project and I think it's important (and just nice) to be familiar with the work of your colleagues. Anyway, I must admit that I find this type of work very dry and just as these authors find my work too descriptive, I find much of this too reductionist. I got tripped up in the second and third sentences of their abstract:

> The influence of yeast mannan on the ecology of the human microbiota is unknown. Here we show that yeast a-mannan is a viable food source for the Gram-negative bacterium Bacteroides thetaiotaomicron, a dominant member of the microbiota.

Reading through the paper there was little discussion of the effect that yeast mannan has on the ecology of the human microbiota. It is a mechanism paper seeking to understand how mannan is broken down by one type of bacterium that has received a lot of attention as a model organism. This gets to the next sentence where the authors state that Btheta is a "dominant member of the microbiota". They repeat this sentiment in the second paragraph of the paper; I should note, without a citation. At the end of the paper they describe how they found the genes for mannan degradation in Btheta in 62% of the subjects that were characterized by shotgun metagenomics. I realize this may seem pedantic, but if a set of genes appears in a majority of individuals and the organisms are not necessarily dominant in those subjects, that does not make them a "dominant member of the microbiota". I'll grant that they may be important, but not necessarily "dominant" in the sense that most people think.

With that in mind, I struggle with work that focuses on model organisms to generalize to the broader microbiome or even just other members of their genus. What does it mean if a bug shows up in 62% of people that is responsible for breaking down mannans? The other 38% of people are probably fine when they eat bread. Furthermore, they found these types of genes in other members of the Bacteroidetes, many of which likely co-occur with Btheta. When we study specific organisms, I do think the knowledge gained is relevant and interesting. However, I have difficulty wrapping my head around whether and how we generalize that information to the broader microbial community given the massive functional diversity of bacteria in the microbiota and the patchy distribution of these organisms across human hosts.


Overall: 5
Importance: 6
Innovation: 5
Who should read:
Tags: human microbiome, reductionism, Bacteroidetes

**Comment by nbaxter13** [comment]
Cuskin and a host of collaborators identify and characterize some genetic loci in Bacteroidetes that evolved to break down yeast mannans commonly found in bread, beer, soy sauce, etc.  This paper has lots of data from genomics, enzymology, structural biology, microbiology and mouse experiments, but they manage to make the content and the story really dry. There’s lots of discussion on polysaccharide utilization loci (PULs),  glycoside hydrolase activity, and cellular localization.  They care a lot more about the biochemistry of carbohydrate degradation and uptake than I ever will.  However, I think their findings are fascinating for several reasons:

1.)	Yeast mannans have been shown to modulate gut immunity and exacerbate IBD.  Degradation of these mannans by gut commensals may be important for humans to enjoy beer and soy sauce.  That also makes beer a prebiotic, which is why this paper made headlines. 
2.)	I shared a lot of beer and science discussion with Martens, Gilbert and his lab at Beneficial Microbes last year.  I think the specificity of these PULs for yeast mannans and not human mannans was a surprise to a lot of people on this paper.  They expected these yeast mannans to block the enyzmes that break down host mannans, which might have had interesting effects on the ecology of the mucus layer.  Instead the enzymes are very specific to yeast mannans.
3.)	The fact that an entirely new system evolved twice (in *Bacteroides* and *Parabacteroides*) for something we started eating not that long ago is really cool.  I’d be interested to see how specific this system is for human isolates, since yeast isn’t a big of a part of other mammals’ diets.  That makes this a really cool example of coevolution that may have been driven exclusively by a change in host diet.  And not just any diet, beer.
4.)	*Bacteroides* evolved to eat beer, making them that much more personable and worth studying.  

I disagree with Pat on downplaying the importance of these findings because they only found these loci in 62% of people.  Do 62% of people have C. diff or colon cancer?  Plus if you look more closely at supplemental figure 8, I think that 62% is an underestimate.  In the groups with the highest sequencing coverage (North Ameraicans) mannan degrading PULs were detected in >80% of samples.  In the groups that had less coverage it was detected in ~50%.  100Mb of coverage means means that *B. theta*  would have to be about 7% of the community to get 1x coverage of its genome.  It’s hard detect a single locus with that kind of coverage.  I’m guessing that 62% would be a lot higher if they dug deeper.   

The authors really focus on the nitty gritty enzymatic details, when they could have been talking about the cool ecology and evolution side of things. Had these enzymes been for anything other than beer, the paper probably wouldn’t have been in Nature or picked up by the media. 

Tl;dr – Beer is a prebiotic.  Drink up!

Overall: 7
Importance: 7
Innovation: 7
Who should read:
Tags: Bacteroides, diet, beer
